crawler.js

```diff
@@ -433,15 +459,23 @@ class Crawler {
       console.warn("Link Extraction failed", e);
       return;
     }
-
     this.queueUrls(results);
+
+    try {
+      var jsonl = ''
+      for (const page of this.pages){
+        jsonl = jsonl + JSON.stringify(page) + "\n"
+      }
+      fs.writeFileSync('pages/pages.jsonl', jsonl)
```

Comment:
The pages directory should be in the same directory as the collection, ideally, eg. next to `archive/`, and should also be created if it doesn't exist..

Reply:
As per above comment, should probably append to an existing pages.jsonl if it already exists, otherwise create new and write the header